qualang_tools/video_mode/README.md

```diff
@@ -0,0 +1,185 @@
+# Video Mode
+## Introduction
+
+In some cases, one might want to update parameters of the program while it is running and without knowing in advance when and how to update them.
+We present here the ```VideoMode``` which enables such dynamic parameter tuning.
+
+## How to use it
+
+The user can declare a ``VideoMode`` instance by passing in a ```QuantumMachine``` instance and a dictionary of
+parameters intended to be updated dynamically. Additionally, if a job is already started on the ```QuantumMachine```,
+it can also use this job to change dynamically some of its parameters (this assumes that the QUA program associated to this job
+was already designed to be compatible with the ```VideoMode```, which we will explain how to do below).
```

Comment:
Is this explained?

Reply:
This features is now removed as it eventually did not make sense to support it for the use case at hand.